cmd/swagger/commands/generate/shared.go

```diff
@@ -75,6 +75,7 @@ type shared struct {
 	ServerPackage         string         `long:"server-package" short:"s" description:"the package to save the server specific code" default:"restapi"`
 	ClientPackage         string         `long:"client-package" short:"c" description:"the package to save the client specific code" default:"client"`
 	Target                flags.Filename `long:"target" short:"t" default:"./" description:"the base directory for generating the files"`
+	Template              string         `long:"template" description:"Load contributed templates" choice:"stratoscale"`
```

Comment:
I would prefer 'TemplateSet' here, I think it is more explicit about the intent.

Reply:
We agree this is a choice string, not a flag, as discussed in the associated issue.

Reply:
is this still relevant?

Reply:
ok let it be like this